README.md

```diff
@@ -221,6 +221,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 - [Geekmarks](https://geekmarks.dmitryfrank.com/) - Personal bookmarking service focused on speed and organization using hierarchical tags. ([Source Code](https://github.com/dimonomid/geekmarks)) `BSD-2-Clause` `Go`
 - [golinks](https://github.com/prologic/golinks) - Web application that allows you to create smart bookmarks, commands and aliases by pointing your web browser's default search engine at a running instance. Similar to bunny1 or yubnub. ([Demo](https://search.mills.io)) `MIT` `Go`
 - [Lobsters](https://lobste.rs) - Run your own link aggregation site. ([Source Code](https://github.com/jcs/lobsters)) `BSD-3-Clause` `Ruby`
+- [Nextcloud Bookmarks](https://github.com/nextcloud/bookmarks) - Bookmarks app for Nextcloud. ([Clients](https://github.com/nextcloud/bookmarks#third-party-clients)) `AGPL` `PHP`
```

Comment:
On a second thought please change `AGPL` to `AGPL-3.0`
See https://github.com/Kickball/awesome-selfhosted#list-of-licenses